tests/resources/validation_reports/validation_output/other_validations/metadata_validation.txt

```diff
@@ -26,4 +26,19 @@
 	should have required property 'bioSampleObject'
 /sample/0
 	should match exactly one schema in oneOf
+/sample/3/bioSampleObject/name
+        must have required property 'name'
+        must have required property 'name'
+        must have required property 'name'
```

Comment:
not sure what's going on here, but this is the output I get from biovalidator...